lib/salestation/app/input_verification.rb

```diff
@@ -10,7 +10,7 @@ def verify_input(schema)
             request.replace_input(result.output)
           else
             Deterministic::Result::Failure(
-              Errors::InvalidInput.new(errors: result.errors, hints: result.hints, debug_message: result.debug_message)
+              Errors::InvalidInput.new(errors: result.errors, hints: result.hints, debug_message: 'invalid_input')
```

Comment:
that's not really helpful. can't we omit debug_message here?

Reply:
sure